lib/puppet_x/puppetlabs/strings/yard/code_objects/defined_type_object.rb

```diff
@@ -1,7 +1,22 @@
+require 'json'
+
 class PuppetX::PuppetLabs::Strings::YARD::CodeObjects::DefinedTypeObject < PuppetX::PuppetLabs::Strings::YARD::CodeObjects::PuppetNamespaceObject
   # A list of parameters attached to this class.
   # @return [Array<Array(String, String)>]
   attr_accessor :parameters
   attr_accessor :type_info
 
+  def to_s
+    require 'pry'; binding.pry
```

Comment:
Looks like we have a bunch of remaining debug statements we should take out (I think these are what's causing the test failures in Travis)